ratelimit/backends/redis.py

```diff
@@ -35,14 +35,14 @@ def __init__(self, redis: StrictRedis) -> None:
         self._redis = redis
         self.lua_script: Script = self._redis.register_script(SCRIPT)
 
-    async def set_block_time(self, user: str, block_time: int) -> None:
-        await self._redis.set(f"blocking:{user}", 1, block_time)
+    async def set_block_time(self, user: str, method: str, block_time: int) -> None:
+        await self._redis.set(f"blocking:{user}:{method}", 1, block_time)
```

Comment:
Block user access to all APIs, not a single API.

Reply:
fixed!